projects/plugins/crm/tests/php/bootstrap.php

```diff
@@ -79,12 +79,22 @@
  * Load Jetpack CRM.
  */
 function _jpcrm_manually_load_plugin() {
+	// Load the main plugin file
 	require_once JETPACK_CRM_TESTS_ROOT . '/../../ZeroBSCRM.php';
 
-	// Run all register_activation_hook() functions.
+	// For tests, we need to manually initialize the plugin
 	global $zbs;
-	$zbs->install();
-	zeroBSCRM_notifyme_createDBtable();
+	if ( class_exists( 'zeroBSCRM' ) && ! isset( $zbs ) ) {
```

Comment:
Why do we need this check? We literally load the class file and we know it's not set. :^)